### ISSUE TITLE
Console output while debugging python

### ISSUE THREAD

**Issue by kristianpenno** [eranif/codelite#3521]
I have a file system workspace setup for python development.
All the expected functionality works as expected.
Except when I launch the debugger, I don't see the output from my script printed anywhere.
I expected it to be in the Output pane.
Is there a window I need to make visible?

![image](https://github.com/user-attachments/assets/23b472e4-3ab4-4930-afdb-a5dc45c551f6)

**Comment by eranif** [comment]
Some background:

CodeLite uses "DAP" (Debug Adapter Protocol) for connecting to debuggers. i.e. there is a "on plugins fits them all".
"debugpy" is no different in this regard: it is an implementation of the DAP protocol.

Having written that, I created 2 sample programs:
- Python
- C++

I debugged the first one using `debugpy` as you did, and indeed, no output.
I debugged the second one using `lldb-dap` (C++ debugger) -> but this time there is an output.

The communication between CodeLite and the debugger is done using "JSON/RPC" protocol (similar to HTTP with JSON payload). I checked CodeLite's log "dap.log" and observed that when debugging in C++, I get these messages (they are initiated from the debugger -> editor):


```json
[07:32:30:848 T:440 DEBUG] dap> <-- {"body":{"category":"stdout","output":"a world\n"},"event":"output","seq":0,"type":"event"}
```

while these type of events are missing when using debugpy.
So this brings me to the conclusion:

- There is some kind of missing setup for debugpy (maybe some kind of a configuration value that needs be passed to debugpy in order to instruct it to send these type of events)
- A bug in debugpy

I will keep on checking...
